go/libkb/leveldb_cleaner.go

```diff
@@ -160,14 +167,24 @@ func (c *levelDbCleaner) clearCache() {
 	c.cache.Purge()
 }
 
+func (c *levelDbCleaner) Shutdown() {
+	c.cacheMu.Lock()
+	defer c.cacheMu.Unlock()
+	var err error
+	c.cache, err = lru.New(1)
```

Comment:
Not sure if this is ok. I can nil this out and make getCache bubble up an error if that'd be better.

Reply:
just log the error instead

Reply:
still need to check if non-nil before using the LRU in that case right?